server/config.js

```diff
@@ -0,0 +1,8 @@
+require('dotenv').config();
```

Comment:
It's a clever and hacky approach, but it may be too clever and too hacky.

1. `process.env` is a kind-of global variable, so using it for storing important value is discouraged.

2. It's always better to have a module with a strongly defined export interface since it will provide excellent code completion in most IDEs.

Reply:
I modify that strategy just because it's easy to test.